docs/maintainers/issue-management.md

```diff
@@ -74,6 +75,7 @@ The following labels are used on PRs:
 * `Review - Abandoned` : The PR was not updated for significant number of days (the exact number could vary over time).
   Maintainers should look into such PRs and re-evaluate them.
 * `Review - Committee` : The PR/Issue needs a review from [powershell-committee](../community/governance.md#powershell-committee)
+* `Review - Need Documentation` : The PR has changes that require a documentation change or new documentation added to [PowerShell-Docs](http://github.com/powershell/powershell-docs)
```

Comment:
Wouldn't this be "Needs"? If you think that's pedantic, whatever, I'm fine with what we've got, but it sounds funny to me.